cmd/buck/cli/roles.go

```diff
@@ -68,7 +74,16 @@ Access roles:
 		if len(args) > 1 {
 			pth = args[1]
 		}
-		res, err := buck.PushPathAccessRoles(ctx, pth, map[string]buckets.Role{args[0]: role})
+
+		identityOrToken := args[0]
+		if err := buckets.ValidateAccessRoleKey(identityOrToken); err != nil {
+			// we will use to identity as raw token, so we compute the hash
```

Comment:
Yes, the idea is that the user of the cli `grant` command would pass in the raw invite token itself (the same token he would use to accept the invite), so that is why I'm doing the hashing of the token here.

I felt this way it would be easier to use the cli to invite and accept without having the cli user needed to explicitly perform the hashing itself. Of course, the grpc function still expects an hashed value.

Let me know if I should force the user to pass in the hash of the token instead for this cli and I'll make the update.